base-notebook/Dockerfile

```diff
@@ -76,14 +76,14 @@ RUN mkdir /home/$NB_USER/work && \
 
 # Install conda as jovyan and check the md5 sum provided on the download site
 ENV MINICONDA_VERSION=4.8.2 \
```

Comment:
There is a new version `4.8.3` (`Miniconda3-py38_4.8.3-Linux-x86_64.sh`) in the [miniconda repo](https://repo.anaconda.com/miniconda/). We should also use this new version.

Reply:
I was trying to limit the change set to avoid breaking things, I figured once this PR gets merged then I can do it in the next PR, unless disagree.

Reply:
Ok it makes sense. Thanks.